libs/vault/src/cipher-form/components/sshkey-section/sshkey-section.component.ts

```diff
@@ -100,6 +114,92 @@ export class SshKeySectionComponent implements OnInit {
     });
   }
 
+  private async importUsingSdk(key: string, password: string): Promise<SshKey> {
+    await firstValueFrom(this.sdkService.client$);
+    return import_ssh_key(key, password);
+  }
+
+  async importSshKeyFromClipboard() {
+    const key = await this.platformUtilsService.readFromClipboard();
+
+    let isPasswordProtectedSshKey = false;
+
+    let parsedKey: SshKey = null;
```

Comment:
⛏️ (non-blocking) small change for when we enable strict TS on this file
```suggestion
    let parsedKey: SshKey | null = null;
```